src/main/java/org/jabref/gui/PreviewPanel.java

```diff
@@ -351,7 +354,14 @@ public void close() {
     }
 
     private void copyPreviewToClipBoard() {
-        String previewContent = (String) previewView.getEngine().executeScript("document.documentElement.outerHTML");
-        clipBoardManager.setContent(previewContent);
-    }
+        StringBuilder previewContent = new StringBuilder();
```

Comment:
So the [java documentation](https://docs.oracle.com/javafx/2/api/javafx/scene/input/Clipboard.html) tells us
> Because multiple representations of the same data may exist on the clipboard, and because different applications have different capabilities for handling different content types, it is important to place as many data representations on the clipboard as is practical to facilitate external applications.

Thus, I would propose to add the html as well as the cleaned text to the clipboard:
````java
     ClipboardContent content = new ClipboardContent();
     content.putString(previewContent.toString());
     content.putHtml(the html content as it was before);
     clipBoardManager.setContent(content);
````